src/test/scala/za/co/absa/springdocopenapiscala/OpenAPIModelRegistrationSpec.scala

```diff
@@ -171,6 +205,29 @@ class OpenAPIModelRegistrationSpec extends AnyFlatSpec {
     assertTypeAndFormatAreAsExpected(actualSchemas, "Maps.b", "object")
   }
 
+  it should "make simple Enumeration (without renaming) an OpenAPI enum" in {
+    val components = new Components
+    val openAPIModelRegistration = new OpenAPIModelRegistration(components)
+
+    openAPIModelRegistration.register[SimpleEnums]()
+
+    val actualSchemas = components.getSchemas
+
+    assertEnumIsStringAndHasFollowingOptions(actualSchemas, "SimpleEnums.a", Set("Pizza", "TV", "Radio"))
+    assertEnumIsStringAndHasFollowingOptions(actualSchemas, "SimpleEnums.b", Set("Cleaner", "Floor"))
+  }
+
+  it should "make complex Enumeration (with renaming) an OpenAPI enum" ignore { // TODO: #23
```

Comment:
What todo is that?

Reply:
This test is ignored as it currently doesn't pass (this PR adds support only for simple Enumerations). Changed the comment to be more descriptive